plugins/permission-backend/src/service/router.ts

```diff
@@ -188,7 +188,13 @@ export async function createRouter(
       req: Request<EvaluatePermissionRequestBatch>,
       res: Response<EvaluatePermissionResponseBatch>,
     ) => {
-      const user = await identity.getIdentity({ request: req });
+      let user;
+      try {
+        user = await identity.getIdentity({ request: req });
```

Comment:
Actually I think we want to make sure that `getIdentity` itself throws a `AuthenticationError` error instead which will be translated to a 401 so there's no need the try/catch :)

So to be clear this should probably be updated instead and 🤞 that it works as expected then; https://github.com/backstage/backstage/blob/master/plugins/auth-node/src/DefaultIdentityClient.ts#L89

Reply:
Oh okay, so the problem is that getIdentity doesn't throw the right error; the permissions router is fine as it is?